src/app/state/staking/saga.ts

```diff
@@ -84,7 +84,14 @@ function* loadDebondingDelegations(publicKey: Uint8Array) {
 
 function* refreshValidators() {
   const { accounts } = yield* call(getExplorerAPIs)
-  const validators = yield* call([accounts, accounts.getValidatorsList], {})
+  let validators
+  try {
+    validators = yield* call([accounts, accounts.getValidatorsList], {})
+  } catch (e) {
+    console.error('get validators list failed, continuing without updated list.', e)
```

Comment:
this would be our first use of console in production. please discuss

we could alternatively build more comprehensive error rendering :shrug:

Reply:
third option: launch the error into the sun

Reply:
How about reporting errors with `put()` method https://redux-saga.js.org/docs/basics/ErrorHandling/ ? I imagined we should show a time-limited popup, if such call fail for example.

Reply:
there's actually a `put` right under this https://github.com/oasisprotocol/oasis-wallet-web/pull/413/files#diff-2fbfbfd97023b036563132e6139c482756aa57759d11641920c1020d9e98391cR43-R47

this log is in addition to the normal user-facing error handling done that way